chess/__init__.py

```diff
@@ -3660,25 +3660,25 @@ def __len__(self) -> int:
     # MutableSet
 
     def add(self, square: Square) -> None:
-        """Adds a square to the set."""
+        """Adds a square to a set."""
```

Comment:
This refers to *the* self object, not just any.

Reply:
Ah, I see. So don't apply this particular change.